jina/peapods/runtimes/asyncio/rest/app.py

```diff
@@ -195,7 +239,15 @@ async def on_connect(self, websocket: WebSocket) -> None:
             logger.success(f'Client {self.client_info} connected to stream requests via websockets')
 
         async def handle_receive(self, websocket: WebSocket, close_code: int) -> None:
+            """
+            Await a message on :meth:`websocket.receive()`
+            Send the message to zmqlet via :meth:`zmqlet.send_message()` and await
+
+            :param websocket: WebSocket connection between clinet sand server.
+            :param close_code: close code
+            """
```

Comment:
Maybe add what's returned here?

Reply:
Seems no return(None) for this function？